scaling_performance/optimizing_storage.adoc

```diff
@@ -11,25 +11,185 @@
 
 toc::[]
 
-== Optimizing Storage
+This topic provides information for optimizing {product-title} storage.
+
+[[general-storage-guidelines]]
+== General Storage Guidelines
+
+The following table lists the persistent storage technology that {product-title} can be used with.
+
+.Available storage options
+[cols="1,4,3",options="header"]
+|===
+| Storage type | Description | Examples
+
+| Block
+| This type of storage is presented to the operating system as a block device. It is
+suitable for applications that need full control of storage and operate at a low level on files
+bypassing the file system. It is also referred to as a Storage Area Network (SAN). It is
+usually a non-shareable type of storage, which means that only one client at a time can
+mount an endpoint of this type.
+| CNS/CRS GlusterFS footnoteref:[disclaimer, CNS/CRS GlusterFS, Ceph RBD, OpenStack Cinder, AWS EBS, Azure Disk, GCE persistent disk, and VMware vSphere support dynamic persistent
```

Comment:
Are footnotes common in these books? If not I'd use Notes not Footnotes.  Footnotes are a very academic paper thing and not common in technical documentation.